schemaspy/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM java:8u111-jre-alpine
```

Comment:
We should probably use `adoptopenjdk/openjdk8:alpine-jre` instead, or `adoptopenjdk/openjdk8:jre8u212-b03-alpine` if we want to be super strict, instead. The "official"  Java / OpenJDK images are a bit shady, they're based on the Debian packages which do not follow the real OpenJDK releases (see [this mail](https://mail.openjdk.java.net/pipermail/jdk8u-dev/2019-May/009330.html) for example), and nobody's really sure who's providing them.